connections/etuovi/etuovi_mapper.py

```diff
@@ -63,7 +63,7 @@ def map_price(
     are saved as cents in ElasticSearch so convert the value to Euros.
     """
     elastic_value = getattr(elastic_apartment, field_name, None)
-    if elastic_value:
+    if elastic_value is not None:
         return convert_price_from_cents_to_eur(elastic_value)
     else:
         return None
```

Comment:
This can be simplified into just a return statement now since the only other case will be `None`.

Reply:
True dad